lib/ProductOpener/Display.pm

```diff
@@ -1624,7 +1638,7 @@ sub display_list_of_tags($$) {
                 text: '$request_ref->{title}'
             },
             subtitle: {
-                text: '$Lang{data_source}{$lc}$sep: @{[ format_subdomain($subdomain) ]}'
+                text: '$Lang{data_source}{$lc}$sep: @{[ $format_subdomain ]}'
```

Comment:
I /think/ the `@{[...}]` syntax was only required, because it called the sub. Shouldn't this work (and be more readable)?
```suggestion
                text: '$Lang{data_source}{$lc}$sep: $format_subdomain'
```